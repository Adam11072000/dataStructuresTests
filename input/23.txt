init 5072
setLabelScore 9986 -9 1000 71
resetLabelScore 5534 2326 426
addImage 6739
addImage 6739
addImage 5097
addImage 6739
setLabelScore 1216 30 -9 7439
quit