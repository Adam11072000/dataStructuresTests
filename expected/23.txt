init done.
setLabelScore: INVALID_INPUT
resetLabelScore: FAILURE
addImage: SUCCESS
addImage: FAILURE
addImage: SUCCESS
addImage: FAILURE
setLabelScore: INVALID_INPUT
quit done.
